_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,303 @@
+---
+title: 'Bitcoin Optech Newsletter #109'
+permalink: /en/newsletters/2020/08/05/
+name: 2020-08-05-newsletter
+slug: 2020-08-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the new Minsc policy language and contains our regular sections with recently
+transcribed talks and conversations, releases and release candidates,
+and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **New spending policy language:** Nadav Ivgi [announced][ivgi minsc] a new
+  language he's developed named Minsc which makes it easier for
+  developers to specify a set of conditions that must be fulfilled in
+  order for a UTXO to be spent.  The new language is based on the
+  [miniscript][topic miniscript] policy language but adds the ability to
+  use variables and functions along with several other features.  Minsc
+  policies can be compiled to miniscript policies, which can be themselves
+  be compiled into miniscript and used to produce regular Bitcoin Script.  Compatibility
+  with miniscript means that policies developed using Minsc will be
+  solvable using any miniscript-aware wallet in the future, allowing
+  wallets to contribute signatures, preimages, or other data needed to
+  spend bitcoins without their developers having to manually specify how
+  to handle each specific script template the wallet supports.  This
+  should allow faster upgrades to new language features and greatly
+  simplify development of interoperable wallets for coinjoins, contract
+  protocols, shared coin ownership, and other types of desirable
+  collaborations.
+
+    Ivgi has also created an outstanding [website][min.sc] for the language.
+    It includes both a plethora of examples and a live compiler that allows
+    linking to its input so that developers can easily play with the
+    language and share their Minsc policies with other developers.  We
+    recommend anyone interested in developing spending policies visit the
+    website but, as an illustration of what Minsc can do, we offer the
```

Comment:
nit
```suggestion
    website, but as an illustration of what Minsc can do, we offer the
```

Reply:
@jonatack and I discussed this same construction a few weeks ago and I just rewrote the sentence to avoid the problem; however, this time I looked it up and it appears I'd bee applying the rule backwards for years. :facepalm:  Thanks to both of you for setting me straight.